packages/next/client/image.tsx

```diff
@@ -684,6 +685,10 @@ function cloudinaryLoader({
   return `${root}${paramsString}${normalizeSrc(src)}`
 }
 
+function unoptimizedLoader({ src }: DefaultImageLoaderProps): string {
+  return `${normalizeSrc(src)}`
```

Comment:
```suggestion
  return src;
```

I think this is just a pass-through, right? You would probably want to keep the leading `/`